### ISSUE TITLE
[4.1] Column Wrapper - Anchor in first column triggers modal before link.

### ISSUE THREAD

**Issue by pxpm** [Laravel-Backpack/CRUD#2839]
# Bug report

Like title describes, when you have a responsive table (Some columns are hidden), if you have an anchor in the first column, when you click it, first it will trigger the modal with the extra columns and only after it will redirect you.

### What I did

Added wrapper href to the first column in table and have some columns hidden.

### What I expected to happen

If the clicked element is an href trigger href, or only trigger the modal if we click in three dots, and not in all first column.

### What happened

It triggers the responsive modal before triggering the href process.

### What I've already tried to fix it

I spend countless hours trying to figure out a solution for this, but still unsolved. 

I tried adding some html element with css (Using an svg file) to the 1st column so we can setup `target` on datatables responsive plugin, but was not able to make the technique work on datatables. 
```
		content: url('data:image/svg+xml;utf8,%3Csvg%20width%3D%2210%22%20height%3D%2210%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3CforeignObject%20width%3D%22100%25%22%20height%3D%22100%25%22%20x%3D%220%22%20y%3D%220%22%3E%3Cspan%20class%3D%22table_responsive_trigger%22%3E%3Ci%20class%3D%22la%20la-ellipsis-v%22%3E%3C%2Fi%3E%3C%2Fspan%3E%3C%2FforeignObject%3E%3C%2Fsvg%3E');

```
REF: https://stackoverflow.com/questions/4505093/css-content-property-is-it-possible-to-insert-html-instead-of-text

**Comment by promatik** [comment]
Hi @pxpm just step by to let you know I'm working on a solution for this and I think I found a possible one. I'll create a PR as soon as possible.

**Comment by pxpm** [comment]
Fixed by #2891
